packages/components/src/Avatar/tests/chromatic/assets/index.ts

```diff
@@ -0,0 +1,5 @@
+import FrogTall from "./frog-tall.jpg";
+import FrogWide from "./frog-wide.jpg";
+import Frog from "./frog.png";
+
+export { Frog, FrogTall, FrogWide };
```

Comment:
sorry for being annoying again, but can you replace both front.png image to an image that has no transparent background? it helps to show that the avatar is round, otherwise it's just the image

Reply:
well I could probably just switch to jpg and hope the white background shows up